client/trino-jdbc/pom.xml

```diff
@@ -362,6 +373,10 @@
                                     <pattern>okhttp3</pattern>
                                     <shadedPattern>${shadeBase}.okhttp3</shadedPattern>
                                 </relocation>
+                                <relocation>
+                                    <pattern>io.opentelemetry.instrumentation</pattern>
+                                    <shadedPattern>${shadeBase}.opentelemetry-okhttp-3.0</shadedPattern>
```

Comment:
Java package names can't contain a dash (at least in Java source code ... might be legal in a class file at the JVM level). Let's name this `opentelemetry.okhttp3`.

Reply:
Done, thanks!